src/components/DropZone.tsx

```diff
@@ -1,9 +1,30 @@
 import React from "react";
 import Dropzone from "react-dropzone";
 
+const onFileDrop = (files: File[]): void => {
+  const reader = new FileReader();
+
+  reader.onerror = () => {
+    alert(`The file uploaded is not a valid TradeTrust file, error: ${reader.error}`);
+  };
+
+  reader.onload = () => {
+    try {
+      if (reader.result && typeof reader.result === "string") {
+        JSON.parse(reader.result);
+        alert(`The file ${files[0].name} will be processed`);
+      }
```

Comment:
else alert ?

Reply:
i have added it

Reply:
tradetrust ? what's that ? change to open attestation

Reply:
updated